src/main/resources/peer-review-transform.xsl

```diff
@@ -90,8 +90,13 @@
     </tr>
   </xsl:template>
 
-  <xsl:template match="contrib">
-    <xsl:value-of select="concat(normalize-space(./name/given-names),' ',normalize-space(./name/surname),', Editor')" />
+  <xsl:template match="contrib-group">
+    <xsl:for-each select="contrib">
```

Comment:
If you like you can change this to 
```xslt
  <xsl:template match="contrib">
    <xsl:value-of select="concat(normalize-space(./name/given-names),' ',normalize-space(./name/surname),', Editor')" />
    <xsl:if test="position() != last()">
      <xsl:text>, </xsl:text>
    </xsl:if>
  </xsl:template>
```

and the `select` on line 78 would be `front-stub/contrib-group/contrib`

Reply:
I like!  Gotta remember that `position()` is not just for `for-each`.